config/database.yml

```diff
@@ -6,6 +6,8 @@ development:
   pool: 5
   database: student_insights_development
   host: localhost
+  username: postgres
```

Comment:
We'll need to remove this to merge, but if you can add a Vagrantfile that works with this (or update the database.yml to take an environment variable for Vagrant builds) in a separate PR that'd be most welcome!